### DISCUSSION TITLE
Live reload without ionic project or xcode

### DISCUSSION THREAD

**Discussion by gkiely** [ionic-team/capacitor#4762] [Q&A]
Hi guys, this was my first time trying out Capacitor, I wanted to test the hot/live reloading, I ran the following commands:
```bash
npm install @capacitor/core @capacitor/cli
npm init @capacitor/app
npm i
npm cap sync
npx cap add ios
npm install -g @ionic/cli native-run
ionic cap run ios -l --external 
```
I got the following error message: 
`[ERROR] Sorry! ionic cap run can only be run in an Ionic project directory.`

Error aside, my question specifically, is whether it is on the roadmap to be able to run hot/live reload without ionic and without Xcode/Android studio, or is this something I could submit a PR for.

Thanks.

**Comment by jcesarmobile** [comment]
We don’t plan to add a live reload server, you can use the one you framework provides or any other live reload server.
The only thing capacitor needs is the server url of your live reload server, how the server works is up to you.
